travis-run-tests.sh

```diff
@@ -39,6 +39,13 @@ function should-tests-run-crossdc-adapter() {
     should-tests-run-crossdc-server
 }
 
+function should-tests-run-adapter-tests-authz() {
```

Comment:
@hmlnarik in which part of the build we invoke this function?

Reply:
@abstractj There is some bash "reflection" magic used in the script. If there is function like "should-tests-run-foo", it can be used to skip the group called "foo". See https://github.com/keycloak/keycloak/blob/master/travis-run-tests.sh#L47